precompiles/dispatch-lockdrop/src/lib.rs

```diff
@@ -81,72 +81,41 @@ where
             pubkey
         );
 
-        let target_gas = handle.gas_limit();
         let caller: H160 = handle.context().caller.into();
         let input: Vec<u8> = call.into();
 
-        // 1. Decode the call
-        let call = Runtime::RuntimeCall::decode_with_depth_limit(DecodeLimit::get(), &mut &*input)
-            .map_err(|_| revert("could not decode call"))?;
+        // Record the cost of the call to ensure there is no free execution
+        handle.record_cost(Runtime::GasWeightMapping::weight_to_gas(
+            Weight::from_parts(1_000_000u64, 0),
```

Comment:
This should be `1_000_000_000` if we want to make it excessive.

E.g. this is used for RocksDB right now:
```Rust
pub const RocksDbWeight: RuntimeDbWeight = RuntimeDbWeight {
	read: 25_000 * constants::WEIGHT_REF_TIME_PER_NANOS,
	write: 100_000 * constants::WEIGHT_REF_TIME_PER_NANOS,
};
```
ref_time per nanos is 1000.

Reply:
The comment should be updated to something like "Record a fixed amount of weight to ensure there is no free execution".